modules/@apostrophecms/modal/ui/apos/components/AposDocsManagerToolbar.vue

```diff
@@ -163,6 +167,11 @@ export default {
   },
   methods: {
     computeActiveOperations () {
+      if (this.isRelationship) {
```

Comment:
I think we should clear the batch operations on the manager level rather than put the work in the toolbar.

Reply:
To me, since this code has impact only in the toolbar module it should stay in the concerned module, it makes the AposDocsManager component lighter.

Reply:
I agree with Val, the toolbar is where the batch operations are displayed, logic concerning when not to show them seems reasonable to put here.

Reply:
I forgot we had the `if` check for filters below as well. Sounds good.